app/src/main/java/com/example/musicapp/RetrofitClientInstance.kt

```diff
@@ -6,9 +6,8 @@ import retrofit2.converter.gson.GsonConverterFactory
 object RetrofitClientInstance {
 
     private var retrofit: Retrofit? = null;
-
```

Comment:
Although it is optional to add space between properties, I took this space out to keep it consistent with other classes throughout your project.